consensus/consensus-types/src/block_retrieval.rs

```diff
@@ -31,6 +48,16 @@ impl BlockRetrievalRequest {
     pub fn num_blocks(&self) -> u64 {
         self.num_blocks
     }
+    pub fn target_block_id(&self) -> Option<HashValue> {
+        self.target_block_id
+    }
+    pub fn match_target_id(&self, hash_value: HashValue) -> bool {
+        if let Some(target_block_id) = self.target_block_id {
```

Comment:
this can be simplified to `self.target_block_id.map_or(false, |id| id == hash_value)`